geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/package.scala

```diff
@@ -30,6 +30,9 @@ package object hbase {
     val RemoteDensityProperty      : SystemProperty = SystemProperty("geomesa.hbase.coprocessor.density.enable")
     val RemoteStatsProperty        : SystemProperty = SystemProperty("geomesa.hbase.coprocessor.stats.enable")
     val YieldPartialResultsProperty: SystemProperty = SystemProperty("geomesa.hbase.coprocessor.yield.partial.results")
+    val CoprocessorThreadsProperty : SystemProperty = SystemProperty("hbase.coprocessor.threads", "16")
+    val MaxRangesPerExtendedScanProperty   : SystemProperty = SystemProperty("hbase.ranges.max-per-extended-scan", "100")
+    val MaxRangesPerCoprocessorScanProperty: SystemProperty = SystemProperty("hbase.ranges.max-per-coprocessor-scan", Int.MaxValue.toString)
```

Comment:
should we make these match the other ones and start with geomesa?
```suggestion
    val CoprocessorThreadsProperty : SystemProperty = SystemProperty("geomesa.hbase.coprocessor.threads", "16")
    val MaxRangesPerExtendedScanProperty   : SystemProperty = SystemProperty("geomesa.hbase.ranges.max-per-extended-scan", "100")
    val MaxRangesPerCoprocessorScanProperty: SystemProperty = SystemProperty("geomesa.hbase.ranges.max-per-coprocessor-scan", Int.MaxValue.toString)
```

Reply:
I committed that but I'm actually thinking of reverting. Reason they don't have the `geomesa.` prefix is because the equivalent data store parameters don't as well. e.g.
https://www.geomesa.org/documentation/stable/user/hbase/usage.html#:~:text=hbase.coprocessor.threads

Reply:
yeah, that's been our pattern - e.g. ds param `accumulo.remote.bin.enable` but sys prop `geomesa.accumulo.remote.bin.enable`. i think the idea is that hbase is probably using sys props prefixed with `hbase.` and we don't want to conflict with them. i.e. from a quick look at the hbase docs, `hbase.master.maintenance_mode` is a sys prop they look for.